app/geo/gui/src/ts/twxplore/gui/geo/api/queries/FeaturesQuery.graphql

```diff
@@ -1,6 +1,12 @@
-query FeaturesQuery($limit: Int!, $offset: Int!) {
-    features(limit: $limit, offset: $offset) {
-        label
-        uri
+query FeaturesQuery($aQuery: FeatureQuery!, $aLimit: Int!, $aOffset: Int!){
+	features(limit:$aLimit, offset:$aOffset, query:$aQuery){
```

Comment:
Formatting looks pretty wonky. You may need another plugin to get prettier to do GraphQL:
https://marketplace.visualstudio.com/items?itemName=Prisma.vscode-graphql